idurar/idurar-erp-crm
# Fixed language issue

## Description

This update addresses an issue where changing the language in the translation toolbar didn't reflect in the application settings. To resolve this, adjustments were made to ensure that selecting a language in the toolbar updates the corresponding language parameter in the settings.

## Related Issues

#1031 

## Steps to Test

1. Change language from header
2. Check if it changed in the app settings
3. Change language from app settings
4. Check if it changed in the header

## Screenshots (if applicable)

![image](https://github.com/idurar/idurar-erp-crm/assets/12747343/a97ef144-7fc7-4ad3-880f-87f6a7c4783a)


## Checklist
